frontend/public/components/events.jsx

```diff
@@ -17,8 +17,8 @@ import { Dropdown, ResourceLink, Box, Loading, NavTitle, Timestamp, TogglePlay,
 import { WSFactory } from '../module/ws-factory';
 import { ResourceListDropdown } from './resource-dropdown';
 
-const maxMessages = 500;
-const flushInterval = 500;
+const maxMessages = 100;
+const flushInterval = 100;
```

Comment:
I've thought about this more, and I'd suggest rolling back the max messages change because we have filters. For example, if I want to show all error events in the cluster, a lot won't show up since we're only looking at the last 100.